### PR TITLE
There is no need to do old etag propagation on shares

### PR DESCRIPTION
**Pull request by rullzer**
Etags are calculated differently now and don't require manual updating.

### CODE REVIEW

**Comment by rullzer** [comment]
The intergration test fail because now te potential conflicts are only resolved on fs mount. And we don't do that in the OCS Share API right now.

**Comment by rullzer** [comment]
Ok 2 more to go... which might be a bug that just gets triggered now... lets see...

**Comment by rullzer** [comment]
Review time!
CC: @schiesbn @nickvergessen @icewind1991 @PVince81

File: lib/private/files/cache/wrapper/cachejail.php [hunk 1]
```diff
@@ -295,6 +295,6 @@ public function moveFromCache(\OCP\Files\Cache\ICache $sourceCache, $sourcePath,
 		if ($sourceCache === $this) {
 			return $this->move($sourcePath, $targetPath);
 		}
-		return $this->cache->moveFromCache($sourceCache, $sourcePath, $targetPath);
+		return $this->cache->moveFromCache($sourceCache, $sourcePath, $this->getSourcePath($targetPath));
```
Comment (rullzer) [inline lib/private/files/cache/wrapper/cachejail.php]: @icewind1991 I'm surprised this did not break sooner ;)

**Comment by nickvergessen** [comment]
Ran the test_sharePropagation*.py smashbox tests: All still pass :white_check_mark:

**Comment by rullzer** [comment]
awesome thanks @nickvergessen

**Comment by icewind1991** [comment]
:+1: looks good

**Comment by rullzer** [comment]
@nickvergessen does that count as a :+1:?

**Comment by PVince81** [comment]
@rullzer please rebase to kick CI or confirm that the integration tests pass locally.

**Comment by PVince81** [comment]
Ah yes, good old propagation :+1:

**Comment by rullzer** [comment]
Rebased for CI.

**Pull request merged**